src/main/java/calculator/Calcurator.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import java.util.Arrays;
+
+public class Calcurator {
+
+    public int calcurateString(String expression) {
+        String[] splitStr = expression.split(" ");
+        int result=0;
+        result = toInt(splitStr[0]);
+        for( int i=0; i<splitStr.length -2; i+=2){
+            char operator = operatorStringToChar( splitStr[i+1]);
+            int secondValue = toInt(splitStr[i+2]);
+            result = calcurate( result, operator, secondValue);
+        }
+        return result;
+    }
+
+    public char operatorStringToChar( String operator) {
+        String str = operator.trim();
+        isBlank( str);
+        return str.charAt(0);
+    }
+
+    private int toInt( String str) {
+        isBlank( str);
```

Comment:
메서드마다 유효성 검증을 해야할 필요가 있을까요?
입력을 받았을때 한번만 검증을 한다면 그 이후엔 검증을 안해도 될 것 같아요!
```suggestion
    private int toInt(String str) {
        isBlank(str);
```